tools/Dashboard/DevHome.Dashboard/Views/AddWidgetDialog.xaml.cs

```diff
@@ -336,4 +336,21 @@ private void WidgetCatalog_WidgetDefinitionDeleted(WidgetCatalog sender, WidgetD
             }
         });
     }
+
+    private void ContentDialog_SizeChanged(object sender, SizeChangedEventArgs e)
+    {
+        const int ContentDialogMaxHeight = 684;

```

Comment:
Because this is in the view, can't these values be retrieved from the XAML?

Reply:
Yes, I think we can retrieve from resources, but actually the best way (since some of these numbers are the same across dialogs) is centralize them and get them from a central resource dictionary. I'd prefer to do that in a separate change.